test/mysql_handler/Dockerfile.dev

```diff
@@ -2,3 +2,6 @@ FROM secretless-dev
 
 RUN apt-get update && \
     apt-get install -y mysql-server
+
+RUN go get github.com/ajstarks/svgo/benchviz && \
+    go get golang.org/x/tools/cmd/benchcmp
```

Comment:
Missing newline at end of line (EOL). Applies to the rest of the files.
See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline